### ISSUE TITLE
Feedback homework for week 3

### ISSUE THREAD

**Issue by remarcmij** [zaherqasqas/hyf-javascript03#2]
Hi Zaher, here is my feedback on your homework for week 3.

Your application loads and runs OK. It is somewhat responsive, although the use of screen space on a mobile phone leaves to be desired (see image):

![zaher](https://user-images.githubusercontent.com/2788771/38928778-90ed6332-430a-11e8-9be8-450aed46c4b5.png)

Your code looks well organised and well formatted. You have correctly implemented promises and `async`/`await`.

**1.** You didn't implement ES6 classes, but don't worry about it. It will be covered again in the React module, which is still some time away.

**2.** You did not sort the list repos displayed in the select box as was asked in Step 4 of the homework.

**3.** You _do_ render (network) error messages to the web page, but they could be more user-friendly. Currently only the message 'Not found' is displayed, which is a bit cryptic for the average user.

Bottom line: I know you struggle with learning all this stuff and need to work hard to get results. Still, if you can keep this up and continue to ask for help when needed I think you have a good chance to make it through the rest of the curriculum.
